pkg/query-service/app/metrics/v4/helpers/clauses.go

```diff
@@ -37,6 +37,17 @@ func GroupByAttributeKeyTags(tags ...v3.AttributeKey) string {
 	return strings.Join(groupTags, ", ")
 }
 
+func GroupByAttributeKeyTagsWithoutLe(tags ...v3.AttributeKey) string {
```

Comment:
where is this referenced?

Reply:
This is not needed anymore. I will remove it.